### DISCUSSION TITLE
Change color and size of Cursor/Pen tip

### DISCUSSION THREAD

**Discussion by Drodriguez819** [geovens/gInk#108] [Ideas]
The default cursor can be a little too big at times and the pen tip a little too small. The option to be able to adjust the size of each would be nice. Also , being able to change the color the each would be helpful. As of now the pen tip is hard to use with dark backgrounds as it blends in. Maybe allowing the pen tip to have an inverted color scheme to whatever color it is hovering over would be an easy fix?

**Comment by pubpub-zz** [comment]
you may be interested in having a look on my fork :
https://github.com/pubpub-zz/ppInk
a) you can select the default cursor to be the pen tip but when you depress alt the cursor becomes the arrow cursor
b) you can modify the cursor arrow using a png file that allow you to draw a large cursor, or a cur file where inversed color can be used
